### PR TITLE
Added 630 notes for chapters 1 and 2

### PR DESCRIPTION
**Pull request by KaraHops**
Added 630 notes for chapters 1 and 2, removed unnecessary files that had been accidentally added previously.

### CODE REVIEW

File: CompTIA_A+_2019/CompTIA_A+_2019.json [hunk 1]
```diff
@@ -155,7 +159,12359 @@
...
+
+      ]
+    },
+    {
+      "__type__": "Note",
+      "data": "",
+      "fields": [
+        "What does CPU stand for?",
+        "Central Processing Unit",
+        "8790700144",
```
Comment (thepurplebuffalo) [inline CompTIA_A+_2019/CompTIA_A+_2019.json]: Duplicate?

File: CompTIA_A+_2019/CompTIA_A+_2019.json [hunk 2]
```diff
@@ -155,7 +159,12359 @@
...
+      "note_model_uuid": "bfa36940-2042-11e9-8b6a-acbc3287506b",
+      "tags": [
+
+      ]
+    },
+    {
+      "__type__": "Note",
+      "data": "",
+      "fields": [
+        "What does CPU stand for?",
```
Comment (thepurplebuffalo) [inline CompTIA_A+_2019/CompTIA_A+_2019.json]: Duplicate?

**Comment by thepurplebuffalo** [comment]
It looks like there are a number of potential duplicates:

```
$ grep -A 1 fields CompTIA_A+_2019.json | grep -ve "--" | grep -v fields.: | sort | uniq -c | grep -v "^ *1 *"
      2         "3.3 volts, 5 volts, and 12 volts are what?",
      3         "A computer component can be damaged with a 30-volt electrostatic discharge. How many volts are needed for a person to feel the discharge?",
      2         "A Molex-keyed connector is what?",
      2         "Disk drive and fan motors use which voltage connector from a PSU?",
      3         "For safety, electrical equipment must be what before beginning work?",
      2         "How are components connected to the motherboard?",
      3         "How can a person discharge any static electricity before working on electronics?",
      3         "How can a workspace be grounded to prevent static buildup?",
      2         "How many pins does the main power connector for the motherboard have?",
      3         "In which computers were horizontal cases most popular?",
      2         "Solder, sockets, expansion slots, and ports are used for what purpose?",
      2         "The 4- to 8-pin auxiliary power connector powers what?",
      2         "The 6- to 8-pin PCIe connector is used to supply power to what?",
      2         "The 'backbone' of a computer is which component?",
      2         "The Berg-keyed connector is what?",
      2         "The original AT power supply form factor did not support which voltage connector?",
      3         "To avoid electrostatic discharge, how should components be stored?",
      3         "What acronym is used for Random Access Memory?",
      2         "What acronym is used for the 'Power Supply' of a PC?",
      2         "What are alternate names for the motherboard?",
      3         "What are alternate terms for a computer case?",
      3         "What are alternative terms for a compact tower?",
      2         "What are the most common voltages supplied by different PSU connectors?",
      3         "What are the original AT and ATX power supply form factors now that newer form factors are more common?",
      2         "What are the primary advantages and disadvantages of a multi-rail PSU vs a single-rail PSU?",
      2         "What biological component of humans is analogous to the motherboard of a PC?",
      3         "What component can damage equipment when swapped between devices with different requirements?",
      2         "What component do the terms 'system board' and 'main board' refer to?",
      3         "What do grounded floor mats, grounded workbench mats, and antistatic wrist straps do?",
      3         "What does a power supply (PSU) do?",
      2         "What does a SATA power connector power?",
      3         "What does ATX stand for?",
      3         "What does CPU stand for?",
      3         "What does ESD stand for?",
      3         "What does RAM stand for?",
      3         "What does the acronym HTPC stand for?",
      2         "What does the acronym 'PSU' stand for?",
      2         "What does the acronym 'RAM' stand for?",
      3         "What does the term 'form factor' refer to?",
      3         "What form factor do the alternate terms 'mini tower' and 'small form factor (SFF)' refer to?",
      3         "What is a device which takes an input, processes it, and provides an output?",
      2         "What is a 'keyed' connector?",
      2         "What is a motherboard?",
      2         "What is a PSU rail?",
      2         "What is an electronic 'bus'?",
      3         "What is the acronym for Advanced Techology Extended?",
      3         "What is the acronym for Home Theatre PC?",
      3         "What is the acronym used for Electrostatic Discharge?",
      2         "What is the acronym used for Random Access Memory?",
      3         "What is the acronym used for the Central Processing Unit?",
      3         "What is the ATX12V power supply form factor in comparison to the other form factors available?",
      3         "What is the most common power supply form factor today?",
      2         "What is the name for an electrical pathway that connects electronic components",
      2         "What is the name for the printed circuit board (PCB) inside a PSU to which external cables are connected?",
      3         "What is the name for the process of discharging static electricity before working on electronics?",
      2         "What is the physical difference between a single and multiple-rail power supply?",
      3         "What is the purpose of antistatic bags?",
      3         "What is the purpose of the case?",
      2         "What is the term for a connector only designed to be inserted one way?",
      2         "What kind of PSU has the advantage of more stable power delivery but the disadvantage of being capable of supplying fewer amps?",
      3         "What might occur when power adapters with different specifications are swapped between devices?",
      3         "What must be grounded before beginning work for safety reasons?",
      3         "What occurs when there is a buildup of static electricity?",
      3         "What power supply form factor was originally designed for network servers?",
      3         "What process does self-grounding describe?",
      2         "What PSU connector voltages are typically consumed by digital circuits?",
      3         "What term is a synonym of the terms chassis, cabinet, tower, housing, or box?",
      3         "What term refers to the physical shape and design of a device's exterior?",
      2         "What type of component makes use of the PSU's 3.3 volt and 5 volt connectors?",
      2         "What type of component typically uses one of a PSU's 12-volt connectors?",
      2         "What type of connector is used to power hard drives, solid state drives, and DVD drives?",
      2         "What type of connector powers the CPU and other individual motherboard components?",
      3         "What type of electricity is an electrostatic discharge?",
      2         "What type of PSU has all connectors on one PCB, and what type of PSU has a separate PCB for each connector?",
      3         "What was the EPS12V power supply form factor originally designed for?",
      3         "What, at its most basic level, does a computer do?",
      3         "Where are compact towers commonly found?",
      3         "Where is EPS12V common today?",
      3         "Which component converts AC power to DC power?",
      2         "Which component does the power supply's 20- or 24-pin connector plug into?",
      2         "Which connector is used to supply power to internal components other than the motherboard?",
      3         "Which form factor is commonly found in corporate environments?",
      3         "Which form factor was most popular in early home computers?",
      2         "Which important component of a PC is essentially just a PCB that contains buses that interconnect electronic components?",
      2         "Which legacy connector is smaller than a Molex connector and used to power floppy drives?",
      2         "Which legacy power connector standard used to power components like hard drives and video cards, but is now relegated mostly to accessories?",
      3         "Which part houses the delicate components of a computer?",
      2         "Which power supply form factor does not support 3.3volt connectors?",
      3         "Which power supply form factors are now considered obsolete?",
      3         "Which power supply is common in high-end desktop PCs?",
      3         "While an ESD needs to be 3000 volts to be felt, how many volts are needed to damage a computer component?",
      3         "Why touch a grounded object before working on electronics?",
```

**Comment by thepurplebuffalo** [comment]
Hmm... it may make sense for us to work on a git pre-commit hook to check for duplicates.

Re: https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks

File: CompTIA_A+_2019/CompTIA_A+_2019.json (hunk 1 above)
Comment (KaraHops) [reply CompTIA_A+_2019/CompTIA_A+_2019.json]: Duplicate removed

File: CompTIA_A+_2019/CompTIA_A+_2019.json (hunk 2 above)
Comment (KaraHops) [reply CompTIA_A+_2019/CompTIA_A+_2019.json]: Duplicate removed

**Review by thepurplebuffalo — APPROVED** [review]

**Pull request merged**